File: Angular/src/app/repair-notes/models/repair-note.ts
import { RepairNoteLine } from './repair-note-line';

export class RepairNote {
    public id: string;
    public capturer: string;
    public lines: RepairNoteLine[];

    constructor() {
        this.id = '';
        this.capturer = '';
        this.lines = [];
    }
}

File: tests/BehaviourTests/OpenProduct.ServiceCenter.Api.BehaviourTests/Models/RepairNote.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace OpenProduct.ServiceCenter.Core
{
    public class RepairNote
    {
        public string Id { get; set; }
        public string Capturer { get; set; }
        public RepairNoteLine[] Lines { get; set; }
    }
}
